discord_slash/utils/manage_commands.py

```diff
@@ -235,3 +235,23 @@ def create_choice(value: str, name: str):
         "value": value,
         "name": name
     }
+
```

Comment:
I think another function should be created to do:
```py
funcname(
    allowed_roles: List[int],
    allowed_users: List[int],
    disallowed_roles: List[int],
    disallowed_users: List[int]
) 
``` 
This would make it more user friendly, reducing the amount of repetition required.

Reply:
What should we name this function? 😄